tests/testthat/_snaps/export.md

```diff
@@ -0,0 +1,202 @@
+# key columns repeat with export_as_txt
+
+    Code
+      cat(listing_exp)
+    Output
+      Unique Subject Identifier   Age   Continous Level Biomarker 1
+      —————————————————————————————————————————————————————————————
+        AB12345-BRA-1-id-134      47    6.5                        
+        AB12345-BRA-1-id-141      35    7.5                        
+      
+      Unique Subject Identifier   Age   Continous Level Biomarker 1
+      —————————————————————————————————————————————————————————————
+        AB12345-BRA-1-id-236      32    7.7                        
+        AB12345-BRA-1-id-265      25    10.3                       
+      
+      Unique Subject Identifier   Age   Continous Level Biomarker 1
+      —————————————————————————————————————————————————————————————
+         AB12345-BRA-1-id-42      36    2.3                        
+         AB12345-BRA-1-id-65      25    7.3                        
+
+# key columns repeat with pagination with export_as_txt
+
+    Code
+      cat(listing)
+    Output
+       Species     Sepal.Length   Sepal.Width   Petal.Length   Petal.Width
+      ————————————————————————————————————————————————————————————————————
+        setosa         5.1            3.5           1.4            0.2    
+                       4.9             3            1.4            0.2    
+                       4.7            3.2           1.3            0.2    
+                       4.6            3.1           1.5            0.2    
+                        5             3.6           1.4            0.2    
+                       5.4            3.9           1.7            0.4    
+                       4.6            3.4           1.4            0.3    
+                        5             3.4           1.5            0.2    
+                       4.4            2.9           1.4            0.2    
+                       4.9            3.1           1.5            0.1    
+                       5.4            3.7           1.5            0.2    
+                       4.8            3.4           1.6            0.2    
+                       4.8             3            1.4            0.1    
+                       4.3             3            1.1            0.1    
+                       5.8             4            1.2            0.2    
+                       5.7            4.4           1.5            0.4    
+                       5.4            3.9           1.3            0.4    
+                       5.1            3.5           1.4            0.3    
```

Comment:
hi @ayogasekaram and @Melkiades , thanks for looking into this again. but the test is still need work

We would like to see horizontal pagination tested here. i.e. the snapshot should like 

![image](https://github.com/insightsengineering/rlistings/assets/3692541/352e6935-4c28-4e8a-b2ba-e248b586d990)